src/org/gk/gkCurator/GKCuratorFrame.java

```diff
@@ -384,9 +384,9 @@ else if (e.getPropertyName().equals("switchType")) {
 	                    GKInstance instance = (GKInstance) e.getNewValue();
 	                    GKSchemaClass oldCls = (GKSchemaClass) e.getOldValue();
 	                    schemaView.switchedType(oldCls, instance);
-	                    eventView.switchedType(oldCls, instance);
```

Comment:
Please see my other comment and keep these lines as they are. Yes, delete one extra line of entityView.switchedType().

Reply:
Okay, the extra line has been removed: https://github.com/reactome/CuratorTool/pull/14/commits/d6d55c0af796c78fe445868c70f67d5d91acef4a